### ISSUE TITLE
The dataset format used for the segmentation task？

### ISSUE THREAD

**Issue by HDL-YD** [itsprakhar/Simple-DinoV2-Classification#5]
I noticed the code about data loading

dataset = SegmentationDataset(img_dir=r"data\segmentation\train\imgs", mask_dir=r"data\segmentation\train\labels", num_classes = 5, img_transform=img_transform, mask_transform=mask_transform)
train_imgs, valid_imgs = train_test_split(dataset.images, test_size=0.2, random_state=42)
train_dataset = SegmentationDataset(img_dir=r"data\segmentation/train/imgs", mask_dir=r"data\segmentation/train/labels", num_classes = 5, img_transform=img_transform, mask_transform=mask_transform, images=train_imgs)
valid_dataset = SegmentationDataset(img_dir=r"data\segmentation/val/imgs", mask_dir=r"data\segmentation/val/labels", num_classes = 5, img_transform=img_transform, mask_transform=mask_transform, images=valid_imgs)

But I don't know what the folder format and file naming should be. I need to train on my own dataset.

**Comment by HaoqianSong** [comment]
Have do you solve the problem? I have the same problem and want to know the data format for training Segmentor.
